api/query/cache/backfill/backfill.go

```diff
@@ -127,6 +127,7 @@ func startBackfillMonitor(store shared.Datastore, logger shared.Logger, maxBytes
 	}
 
 	// Start the monitor to ensure that memory pressure is tracked.
+	// nolint:errcheck // TODO: Fix errcheck lint error.
 	go m.Start()
```

Comment:
instead of nolinting this, do something like

```go
go func() {
  err := m.Start()
  if err != nil {
    logger.Warningf("Failed to  start back fill: %s", err.Error())
  }
}()
```